### PR TITLE
Feature c

### PR DESCRIPTION
**Pull request by chinweokwu**
Responsive design project
This is a responsive web design project,  which was created in branch feature-c.  We will use html and css principles to make a clone site like TNW homepage. This project has 3 break point as specified by microverse. we also used flex and grid for the layout of this project.

### CODE REVIEW

**Comment by kobiyoyo** [comment]
## Pull Request under review, by Adama Chubiyojo Desmond (TSE)

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team      
#### Great Job so far, but there are a few more things required. 
## Required Changes:
#### General 
- [x] Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history
- [x] Pull Request has meaningful title 
#### Latest news section
###### Small version changes
- [ ] The images are a bit big compared to original
### Original

### Yours




#### Latest deals section
###### Small version changes
- [ ] The images are a bit big compared to original
### Original



### Yours



##### Footer
* [x] It contains social media icons vertically and horizontally aligned
- [x] Elements should be aligned with [float](https://css-tricks.com/all-about-floats/) and [flex](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) or [grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout)
### Make changes and submit another code review request.
*Happy coding*

File: css/mediaquery.css [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+  }
+
+  .bars {
+    display: flex;
+    width: 180px;
+    height: 45px;
+  }
+
+  .ads {
+    display: block;
```
Comment (kobiyoyo) [inline css/mediaquery.css]: Elements should be aligned with float and flex or grid

**Comment by uryelah** [comment]
Hello **Mauricio Santos Hoyos** & **Morah Morah**, I'm Sarah, your reviewer for this PR .

You did a great job implementing the required changes from the last review.
Congratulations the project is very responsive! 🥇 

The project is almost good to be approved, you just need to fix a little text spacing issue on small screens.

-------

### Required Changes

- [ ]  The text at the bottom of the main sections, starting from apps, has some margin in the original site. Check the comparison between your project and the original site bellow.

Your version:
Screenshot_2020-02-28 Page Title

Original site:
Screenshot_2020-02-28 TNW - International technology news, business and culture(4)

-------

**Kindly implement the changes above and request a new code review.**

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

**Comment by SofiBretz** [comment]
Code under review by SofiBretz. Thanks!✨

**Comment by SofiBretz** [comment]
### Changes required ♻️
### Hello team 🌟

Great work on this project, but you need to fix the issues below before moving on ⬇️

- [ ] Consider resize the images as the original page, on the 'Latest Deals' and on 'Latest News' and add a margin between texts.





- [ ] Consider follow the design as the original page, the decoration orange circles have to be inline with the text. 



- [ ] Pull Request has a meaningful title and a short description, [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/
) you can learn about it. 

[OPTIONAL]

- [ ] Consider adding the demo link in your readme file.

_**After making the required changes please submit a new code review request**_
Good luck! 🍀 You are almost there! 💪

SofiBretz.

**Comment by lekegitrepo** [comment]
This **PR** is under Review by **_[TSE] Temitayo Adeleke_**

**Comment by lekegitrepo** [comment]
### Hi Team,
You have done an excellent job. **_This is approved._** Based on making the required changes stipulated by the previous reviewer.

**_However, you can check the images below to improve the page responsiveness at 768px and below breakpoints_**

## Optional:

**_This is your page at 768px:_**

image

**_This is the original at 768px:_**

image

**_This is your page responsive below 768px:_**

image

**_This is the original:_**

image

Reviewed by: Temitayo Adeleke

-   Github: [@lekegitrepo](https://github.com/lekegitrepo)
-   Twitter: [@leketemi](https://twitter.com/leketemi)
-   Linkedin: [linkedin](https://www.linkedin.com/in/adeleke-temitayo-a69125188/)

**Review by lekegitrepo — APPROVED** [review]
## Status: Approved.

**Pull request merged**